### PR TITLE
Create News "2018-10-31-navcoin-core-4-4-0-voting-underway"

### PR DESCRIPTION
**Pull request by kierenhyl**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/news/2018-10-31-navcoin-core-4-4-0-voting-underway.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+There will be announcements in social media when NavCoin Core 4.4.0 has been approved. Once consensus is reached there will be a one week lock-in period and after that, the NavCoin Community Fund will be activated and open to your submissions. You can check the progress of the network consensus at the [NavCoin block explorer](https://www.navexplorer.com/soft-forks).
+
+### Upcoming NavPi image update
+
+The new NavPi image will be a refreshed version of the image that you can download and flash to an SD Card when you’re setting up your NavPi. The new image will include the up-to-date blockchain, NavCoin 4.4.0 and a simplified version of the UI for basic Community Fund interactions. Keep an eye out for this release in the next few days.
+
+### NPIP voting
```
Comment (matt-auckland) [inline content/news/2018-10-31-navcoin-core-4-4-0-voting-underway.md]: Only 2/3 softforks are NPIPs just fyi

Comment (matt-auckland) [inline content/news/2018-10-31-navcoin-core-4-4-0-voting-underway.md]:
```suggestion
### Softfork Voting
```

File: content/news/2018-10-31-navcoin-core-4-4-0-voting-underway.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+[This proposal aims](https://github.com/NAVCoin/npips/blob/master/npip-0003.mediawiki) to save space on the blockchain by having fewer transactions in each block. So rather than depositing money to the Community Fund in small amounts every block, it will be done in a single large amount every 500 blocks. 
+
+### NPIP005 - voting in progress
+
+[The idea behind NPIP005](https://github.com/NAVCoin/npips/blob/master/npip-0005.mediawiki) is to double the amount of NAV that the network contributes to the Community Fund from 0.25 NAV per block to 0.5. It’s hoped that this will result in more proposals being worked on and completed. Check out some of the most recent discussions [in this Reddit thread](https://www.reddit.com/r/NavCoin/comments/9s91j6/version_bit_16_signals_an_increase_to_the_cf?sort=confidence).
+
+### NPIP004 - under review
+
+NPIP004 proposes moving to a static block reward scheme to better incentivise stakers to assist in increasing network security. It’s planned to include this NPIP in the upcoming NavCoin 4.5.0 as a soft fork. Please check it out on GitHub, and feel free to critique it constructively.
```
Comment (matt-auckland) [inline content/news/2018-10-31-navcoin-core-4-4-0-voting-underway.md]:
```suggestion
NPIP004 proposes moving to a static block reward scheme to better incentivise stakers to assist in increasing network security. It’s planned to include this NPIP in the upcoming NavCoin 4.5.0 as a soft fork. Please check it out [on GitHub](https://github.com/NAVCoin/npips/blob/master/npip-0004.mediawiki), and feel free to critique it constructively.
```

File: content/news/2018-10-31-navcoin-core-4-4-0-voting-underway.md (hunk 1 above)
Comment (kierenhyl) [reply content/news/2018-10-31-navcoin-core-4-4-0-voting-underway.md]: Changed

**Pull request merged**